README.adoc

```diff
@@ -17,6 +17,22 @@ This project is focused on bringing a minimal http://github.com/openshift/origin
 
 If you are looking for a solution for running OpenShift 3.x, you will need tools such as `oc cluster up`, http://github.com/minishift/minishift[Minishift] or https://developers.redhat.com/products/cdk/overview/[CDK].
 
+[[usage-data]]
+== Usage data
+
+With your approval, CodeReady Containers collect usage data and send it pseudonymized to Red Hat servers to help improve our products and services. Read our https://developers.redhat.com/article/tool-data-collection[privacy statement] to learn more about it. To know the specific data points being collected, see xref:usage-data.adoc#data-table[usage-data.adoc].
+
+The first time a version of CodeReady Containers is run, you will be asked to opt-in to Red Hat's telemetry collection program:
+
+```bash
+Would you like to contribute anonymous usage statistics? [y/N]:
+```
+
+=== Disable usage data collection
```

Comment:
This isn't actually about disabling -- this section describes manually consenting to data collection. Perhaps "Manually configuring usage data collection"?